### DISCUSSION TITLE
Where to put if else in the code please suggest

### DISCUSSION THREAD

**Discussion by officialpalash** [tttapa/Control-Surface#756] [General]
Where to put if else in the code please suggest
Please help

**Comment by tttapa** [comment]
Hi, for programming questions not directly related to Control Surface, please use https://forum.arduino.cc/ or follow the tutorials at https://learncpp.com/.

**Comment by officialpalash** [comment]
Can i use _if else_ under void loop

**Reply by tttapa** [reply]
Sure, you can use an _if_ statement inside of any function.
